src/itty-fetcher.ts

```diff
@@ -66,13 +68,25 @@ const fetchy =
     }
 
     const full_url = (options.base || '') + url_or_path + search
-    const passthrough = payload instanceof FormData || payload instanceof Blob
+
+    /**
+     * Detect what type of incoming payload is provided. If it is FormData, Blob
+     * or Uint8Array, we will not attempt to stringify it. Otherwise, we will.
+     *
+     * TODO: This is a bit funky and isn't very extensible. We should probably
+     * find a better approach.
+     */
+    const is_formdata = typeof FormData !== 'undefined' && payload instanceof FormData
+    const is_blob = typeof Blob !== 'undefined' && payload instanceof Blob
+    const is_arraybuffer = typeof Uint8Array !== 'undefined' && payload instanceof Uint8Array
+    const passthrough = is_formdata || is_blob || is_arraybuffer
+
```

Comment:
Gulp.

Reply:
Granted this should minify somewhat...

Reply:
I think you can make this smaller by keying on 'raw' post content vs stringify post content:

```ts
fetcher.post('/whatever', raw(myBlobHere))
```

A tiny function `raw` could simply tag the object with a symbol to skip the stringification...

Reply:
I think I figured out a better approach! Using `Object.getPrototypeOf()` we can sniff the type of the object. If it is of type `Object` we can stringify it otherwise we pass through. Think this could work? Feels more robust than what were doing now

![C1C0A4E6-230C-42D7-8A31-1C5E0C94F0EB](https://user-images.githubusercontent.com/157695/196005191-b8cfe188-0a99-4ae0-8665-746fadcc4457.jpeg)